interpreter/src/symbol_table/mod.rs

```diff
@@ -31,7 +31,7 @@
 #![allow(dead_code)]
 
 mod hash;
-mod m_var;
+pub mod m_var;
```

Comment:
Take another look at what needs to be public. 

making the whole modal public is probably over kill.